code/game/objects/items/food/sandwichtoast.dm

```diff
@@ -5,33 +5,36 @@
 	icon_state = "sandwich"
 	trash_type = /obj/item/trash/plate
 	food_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/consumable/nutriment/protein = 7, /datum/reagent/consumable/nutriment/vitamin = 1)
-	microwaved_type = /obj/item/food/toastedsandwich
 	tastes = list("meat" = 2, "cheese" = 1, "bread" = 2, "lettuce" = 1)
 	foodtypes = GRAIN | VEGETABLES
 	w_class = WEIGHT_CLASS_SMALL
 
-/obj/item/food/toastedsandwich
-	name = "toasted sandwich"
-	desc = "Now if you only had a pepper bar."
+/obj/item/food/grilledcheesesandwich
```

Comment:
would you mind throwing some underscores in this since we're touching it anyway

Reply:
> 
> 
> would you mind throwing some underscores in this since we're touching it anyway

Of course, that should fix it.